pkg/csi/service/vanilla/controller.go

```diff
@@ -1157,3 +1157,11 @@ func (c *controller) ListSnapshots(ctx context.Context, req *csi.ListSnapshotsRe
 	log.Infof("ListSnapshots: called with args %+v", *req)
 	return nil, status.Error(codes.Unimplemented, "")
 }
+
+func (c *controller) ControllerGetVolume(ctx context.Context, req *csi.ControllerGetVolumeRequest) (
+	*csi.ControllerGetVolumeResponse, error) {
+	ctx = logger.NewContextWithLogger(ctx)
+	log := logger.GetLogger(ctx)
+	log.Infof("ControllerGetVolume: called with args %+v", *req)
+	return nil, status.Error(codes.Unimplemented, "")
```

Comment:
we do not need to initialize context and logger for Unimplemented interfaces.
we just need to return `return nil, status.Error(codes.Unimplemented, "")`

Reply:
+1

Reply:
Removed. But other unimplemented interfaces in our controller are logging the request. So should we keep it like that to keep it consistent with other interfaces?
For example, https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/cd44e3a2aa12b94cd5d033225f91f8a2018daefc/pkg/csi/service/wcp/controller.go#L781